infra/templates/ado-bootstrap-iac-pipeline/.gitignore

```diff
@@ -0,0 +1,32 @@
+.envrc
```

Comment:
Should we have this in the top level `.gitignore` file?

Reply:
Yes, but I haven't run across it's usage elsewhere. More generally, should all of this files contents be in the root .gitignore for the repo?

Reply:
Probably, though this is based purely on what I've done in the past, and less about any best practice. your call.

Reply:
I migrate the ignores up to the root.